examples/with-next-auth/.env.local.example

```diff
@@ -1,11 +1,13 @@
-GOOGLE_ID=
-GOOGLE_SECRET=
-FACEBOOK_ID=
-FACEBOOK_SECRET=
-TWITTER_ID=
-TWITTER_SECRET=
-GITHUB_ID=
-GITHUB_SECRET=
-EMAIL_SERVER=smtp://username:password@smtp.example.com.com:587
-EMAIL_FROM=NextAuth <noreply@example.com>
-DATABASE_URL=sqlite://localhost/:memory:?synchronize=true
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_TWITTER_ID=
+NEXTAUTH_TWITTER_SECRET=
+NEXTAUTH_GITHUB_ID=
+NEXTAUTH_GITHUB_SECRET=
+NEXTAUTH_GOOGLE_ID=
+NEXTAUTH_GOOGLE_SECRET=
+NEXTAUTH_FACEBOOK_ID=
+NEXTAUTH_FACEBOOK_SECRET=
+NEXTAUTH_EMAIL_SERVER=smtp://username:password@smtp.example.com:587
+NEXTAUTH_EMAIL_FROM=NextAuth <noreply@example.com>
+NEXTAUTH_DATABASE_URL=sqlite://localhost/:memory:?synchronize=true
```

Comment:
What's the reason to put `NEXTAUTH_` as a prefix for all envs?

Reply:
Consistence with upstream: https://github.com/nextauthjs/next-auth/blob/main/.env.example + IMO things like `EMAIL_SERVER`, 'GOOGLE_ID' deserves some context when you include the example in an existing  app.

Reply:
If I were to clone this on my own app and I already have keys for twitter, github, google, facebook, e.t.c I would try to not have the same keys under different names. I like the simplicity of the names it had before.

Reply:
Assumption valid. Might change your mind: I have multiple ZZZ_GITHUB_XXX_ID, ZZZ_GOOGLE_XXX_ID... envs. I guess I'm not the only one. This seems more resilient to add context (at least for things like `EMAIL_FROM`) from the start. Let me know, I can change it back... But my 2 cents: following upstream notation might help when searching doc, issues...

Reply:
Keep it like this. It's up to you and I agree with what you said.

Reply:
Actually I agree with you too... I like simplicity as well. I'll let it like this then. Thanks for review